app/styles/components/_labs.scss

```diff
@@ -0,0 +1,2 @@
+.labs-row .notes { max-width: 300px; }
+.completed-labs-row .notes { max-width: 300px; }
```

Comment:
@cacqw7 saw comment as #437 - not sure why we need two different classes here. @jglovier any opinion on using two different classes for completed vs in-progress?